src/crawlers/cheerio_crawler.js

```diff
@@ -188,6 +169,21 @@ const DEFAULT_AUTOSCALED_POOL_OPTIONS = {
  *   An array of <a href="https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Complete_list_of_MIME_types"
  *   target="_blank">MIME types</a> you want the crawler to load and process.
  *   By default, only `text/html` and `application/xhtml+xml` MIME types are supported.
+ * @property {string} [responseEncoding]
+ *   By default `CheerioCrawler` will extract correct encoding from the HTTP response headers.
+ *   Sadly, there are some websites which use invalid headers. Those are encoded using the UTF-8 encoding.
+ *   If those sites actually use a different encoding, the response will be corrupted. You can use
+ *   `responseEncoding` to fall back to a certain encoding, if you know that your target website uses it.
+ *   ```
+ *   // Will fall back to windows-1250 encoding if none found
+ *   responseEncoding: 'windows-1250'
+ *   ```
+ *   You can also prepend your encoding with an exclamation mark to force use of the selected encoding
+ *   even when `CheerioCrawler` able to extract a valid encoding from the response:
+ *   ```
+ *   // Will use windows-1250 encoding even for responses with a set encoding
+ *   responseEncoding: '!windows-1250'
```

Comment:
This thing with exclamation mark feels like a non-standard hack. When people see that in code, they will have to look into docs to see what that means. Having two properties e.g. `forceResponseEncoding` and `suggestResponseEncoding` would make it pretty clear.

Reply:
Why not.